main/php.h

```diff
@@ -47,6 +47,8 @@
 # define PHP_OS_FAMILY			"Solaris"
 #elif defined(__linux__)
 # define PHP_OS_FAMILY			"Linux"
+#elif defined(__midipix__)
+# define PHP_OS_FAMILY			"Midipix"
```

Comment:
PHP OS family is a very simplistic categorization of operating systems. Probably a bit too simplistic design from the beginning to be useful in PHP code.

There is also PHP_OS constant which has probably "midipix" string on such platform.

Cygwin, MinGW, MSYS, and probably midipix could be categorized as Windows. Is _WIN32 symbol perhaps defined in midipix? Or if we "invent" another category here: UNIX?

Reply:
`_WIN32` is not defined on midipix, `UNIX` would be misleading, `POSIX` perhaps? that would also work for cygwin

Reply:
I see. Yes. Would it be better perhaps that this change is a separate PR because then also PHP docs will need to be updated a bit. And we can recheck it there what would be the good approach here. We need to update this categorization anyway.

Reply:
OK, I've taken that change out:-)

Reply:
Thank you @Redfoxymoon